libs/design-system/src/panda/radius.tokens.ts

```diff
@@ -0,0 +1,45 @@
+import { defineTokens } from '@pandacss/dev';
+
+/**
+ * Represents the size of an element.
+ *
+ * Used for properties like width and height.
+ */
+export const RADIUS_TOKENS = defineTokens.radii({
+  '50': {
+    value: '0.25rem',
+    type: 'radius',
+  },
+  '75': {
+    value: '0.375rem',
+    type: 'radius',
+  },
+  '100': {
+    value: '0.5rem',
+    type: 'radius',
+  },
+  '150': {
+    value: '0.75rem',
+    type: 'radius',
+  },
+  circle: {
+    value: '50%',
+    type: 'radius',
+  },
+  pill: {
+    value: '9999px',
+    type: 'radius',
+  },
+});
+
+/** @deprecated */
+export const LEGACY_RADIUS_TOKENS = defineTokens.radii({
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** These line-up with values that are used frequently throughout web‏